sunpy/map/maphelper.py

```diff
@@ -0,0 +1,246 @@
+    creating a `sunpy.map.Map`.
+
+    Examples
+    --------
+
+    * Print all accepted meta keywords that are used in a `sunpy.map.GenericMap` header:
+        >>> from sunpy import map
+        >>> map.meta_keywords()
+        cunit1  :  Units of the coordinate increments along naxis1 e.g. arcsec **required
+        cunit2  :  Units of the coordinate increments along naxis2 e.g. arcsec **required
+        crval1  :  Coordinate value at reference point on naxis1 **required
+        crval2  :  Coordinate value at reference point on naxis2 **required
+        ...
+
+    * Print information about certain meta keyword:
+        >>> map.meta_keywords('ctype1')
+        ctype1  :  Coordinate type projection along naxis1 of data e.g. HPLT-TAN
+
+    """
+    if key:
+        for k in key:
+            if k in map_meta_keywords.keys():
+                print(k, " : ", acceptable_meta[k])
+            else:
+                print(k, " is not a meta keyword required for sunpy.map.Map \n The map meta keywords are : \n", list(map_meta_keywords.keys()))
+        return
+
+    for k in map_meta_keywords:
+        print(k, " : ", map_meta_keywords[k])
+
+
+@u.quantity_input
+def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None, scale: u.arcsec/u.pix = None, **kwargs):
+    """
+    Function to create a `sunpy.util.meta.MetaDict` from a coordinate object (`astropy.coordinates.SkyCoord` or coordinate frame)
+    that is required to create a `sunpy.map.Map`
+
+    Parameters
+    ----------
+    data : `~numpy.ndarray`
+        Array data of Map for which a header is required.
+    coordinates : `~astropy.coordinates.SkyCoord`
+        Coordinate object to get meta information for map header.
+    reference_pixel :`~astropy.units.Quantity` of size 2, optional
+        Reference pixel along each axis. These are expected to be Cartestian ordered, i.e
+        the first index is the x axis, second index is the y axis.
+        Defaults to the center of data array, ``(data.shape[0] + 1)/2., (data.shape[1] + 1)/2.)``.
+    scale : `~astropy.units.Quantity` of size 2, optional
+        Pixel scaling along x and y axis (i.e. the spatial scale of the pixels (dx, dy)). These are
+        expected to be Cartestian ordered, i.e [dx, dy].
+        Defaults to ``([1., 1.] arcsec)``.
+    **kwargs:
+        Additional arguments that will be put into the metadict header. These must be in the
+        list of map meta keywords. The keyword arguments for the instrument meta can
+        be given as ``instrument``, ``telescope``, ``observatory``, ``wavelength``, ``exposure``,
+        which will be translated to the accepted meta keywords ``instrume``, ``telescop``,
+        ``obsrvtry``, ``wavelnth``, ``exptime`` respectively.
+
+    Returns
+    -------
+    `~sunpy.util.MetaDict`
+        A `sunpy.util.MetaDict` that contains the header information required for making a `sunpy.map.Map`
+
+    Examples
+    --------
+    >>> from sunpy import map
```

Comment:
`map` is a Python builtin:

```suggestion
    >>> import sunpy.map
```